client/components/mma/accountoverview/updateAmount/SupporterPlusUpdateAmountForm.tsx

```diff
@@ -63,7 +63,7 @@ function validateChoice(
 	mainPlan: PaidSubscriptionPlan,
 ): string | null {
 	const chosenOptionNum = Number(chosenAmount);
-	const monthlyOrAnnual = calculateBillingPeriod(
+	const monthlyOrAnnual = calculateBillingPeriodAdjective(
```

Comment:
Nice renaming

Reply:
should we rename the `calculate` bit? maybe just `get` ?

Reply:
yes that's a good idea...

```suggestion
	const monthlyOrAnnual = getBillingPeriodAdjective(
```